test/WasmSpec/convert-test-suite/index.js

```diff
@@ -114,7 +114,7 @@ function main() {
     const runs = specFiles.map(specFile => {
       const ext = path.extname(specFile);
       const basename = path.basename(specFile, ext);
-      const isXplatExcluded = config["xplat-excludes"].indexOf(path.basename(specFile, ext)) !== -1;
+      const isXplatExcluded = true || config["xplat-excludes"].indexOf(path.basename(specFile, ext)) !== -1;

```

Comment:
not really a fan of this, because when we enable crossplat we will need to remember to change this true || ... condition, which we almost certainly will forget to do